tests/queries/0_stateless/02421_json_decimals_as_strings.sql

```diff
@@ -0,0 +1,4 @@
+select toDecimal128(42.42, 5) as d format JSONEachRow settings output_format_json_quote_decimals=1;
+insert into function file(02421_data.jsonl) select '42.42' as d settings engine_file_truncate_on_insert=1;
```

Comment:
Mostly we map just by format name, but in case of `jsonl` it's a standart file extension for such files. But it's a good idea to document it, I will create separate PR with it

Reply:
Sounds good.